### ISSUE TITLE
Statistics: condense rewards for Gbg / Statistik: GG-Belohnungen zusammenfassen

### ISSUE THREAD

**Issue by capshort** [dsiekiera/foe-helfer-extension#962]
I absolutely love this feature. However, it is not always easy to see which type of reward I get how often. E.g. for me it is not relevant _which type_ of unit I got, only the amount of units received (seems to be either one or five). Same thing with the goods, not relevant __which__ I got, only how many (seems to be either 10 or 50). It would be very nice to have an option to condense those same types of rewards with the same amount. Hopefully this would also prevent the same colour being used twice in the chart (since this complicates reading the statistics, see screenshot).

---

Das Feature ist wirklich klasse. Allerdings sehe ich oft erst auf den dritten Blick, welche Arten von Belohnungen ich wie oft bekommen habe. Für mich ist zum Beispiel egal, _welche Art_ von Einheiten ich bekomme, nur die Anzahl ist wichtig (bisher warens immer eine oder fünf). Genau das gleiche mit den Gütern, mir ist egal _welche Sorte_ Güter ich bekomme, nur die Anzahl zählt (waren bisher immer 10 oder 50). Es wär super wenn man einstellen könnte, dass solche Belohnungen zusammengefasst werden. Damit wäre dann hoffentlich auch nicht mehr zweimal dieselbe Farbe im Diagramm enthalten (denn das erschwert das Auslesen der Statistiken, siehe Screenshot).

![Screenshot_20200503_083433](https://user-images.githubusercontent.com/25132855/80907644-f35e5a80-8d18-11ea-841f-0cfbead4b525.png)

**Comment by osv** [comment]
Yes, this is not new idea, some optional toggle button "group by resource" should be implemented.

**Comment by scibuff** [comment]
Another 64/64 week of GE with ToR level 58 stats
[rewards-tor-lvl-58-extra-ge.txt](https://github.com/dsiekiera/foe-helfer-extension/files/4580664/rewards-tor-lvl-58-extra-ge.txt)
